setup.py

```diff
@@ -11,8 +11,8 @@
     description='Recursive Feature Elimination on Spark',
     long_description=long_description,
     long_description_content_type='text/markdown',
-    author='Christopher Baker, Matthew Bowman, Brady Wilkerson',
-    author_email='chriscrewbaker@gmail.com',
+    author='Christopher Baker, Matthew Bowman, Brady Wilkinson',
+    author_email='chriscrewbaker@gmail.com', 'bradywilky@gmail.com'
```

Comment:
You can't specify multiple author emails in setup.py, hence the build is failing.

https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py

Reply:
You can add your email to a the author_email string if you want, but you can't just provide a tuple of strings like that